packages/sdk-metrics/src/aggregator/types.ts

```diff
@@ -49,7 +49,7 @@ export interface Histogram {
    *
    * buckets: {
    *	boundaries: [10, 20, 30],
-   *	counts: [3, 3, 1, 2],
+   *	counts: [3, 3, 2, 1],
```

Comment:
It would be good to mention what spec describe here in the notes maybe, so is clear why the counts are like that. "buckets express the number of values that are greater than their lower bound and less than or equal to their upper bound."